codespell_lib/data/dictionary.txt

```diff
@@ -7738,6 +7747,7 @@ fonetic->phonetic
 fontain->fountain, contain,
 fontains->fountains, contains,
 fontier->frontier
+fontonfig->fontconfig
```

Comment:
Specific

Reply:
Removed. But - as I say - this is "codespell". So it should make sense to leave it.

Reply:
Fair point, see what @larsoner thinks. Personally I think it should probably be in a separate dictionary.

Reply:
No problem. We can discuss about it in the future.

Reply:
I'm fine with keeping the more coding-specific ones so long as they are "sufficiently common". `fontconfig` is pretty common I think

Reply:
Do you want to add this back in then @Gelma ?

Reply:
Ok, thanks.